src/compressed_tensors/entrypoints/convert/converters/fp8block_dequantizer.py

```
@@ -25,6 +30,12 @@ def __init__(
         weight_block_size: tuple[int] = (128, 128),
         dtype=torch.bfloat16,
     ):
+        warnings.warn(
```

Ruff (0.15.12)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)